### ISSUE TITLE
Slack notifications don't work when deploying new image

### ISSUE THREAD

**Issue by Roberdvs** [fabfuel/ecs-deploy#122]
Hi! Thanks for the tool.

I was playing around with the Slack notifications and they work fine when using the -t flag but not when passing images with the -i one.

Looks like tag variable is null over here:
 https://github.com/fabfuel/ecs-deploy/blob/1116ba7d9f7dd256b9668bbcf9cad973da4fde5d/ecs_deploy/cli.py#L87
 
Here's the full stack trace:
```
Traceback (most recent call last):
  File "/usr/local/bin/ecs", line 8, in <module>
    sys.exit(ecs())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/ecs_deploy/cli.py", line 87, in deploy
    slack.notify_start(cluster, tag, td, comment, user, service=service)
  File "/usr/local/lib/python3.7/site-packages/ecs_deploy/slack.py", line 63, in notify_start
    if diff.field == 'image' and diff.value.endswith(':' + tag):
TypeError: can only concatenate str (not "NoneType") to str
```

**Comment by vivekyad4v** [comment]
+1

**Comment by fabfuel** [comment]
I have just released Version 1.10.1 (which includes a fix for this issue) to PyPI
